amethyst_renderer/src/tex.rs

```diff
@@ -46,6 +46,12 @@ impl Texture {
     pub fn view(&self) -> &RawShaderResourceView {
         &self.view
     }
+
+    /// Returns the texture's dimensions ``(width, height)``
+    pub fn size(&self) -> (usize, usize) {
+        let (w, h, _, _) = self.texture.get_info().kind.get_dimensions();
```

Comment:
I'm not entirely sure about this, doesn't this invoke a driver function?

Reply:
nop, not on the gfx-rs side

Reply:
Okay, great thanks.

Reply:
Yeah @msiglreith is correct, this merely traverses structures in RAM.  **Some** computation could be saved by calling this less frequently but this absolutely falls in the category of premature optimization imo.

Reply:
My first approach was to store the size in the Texture struct, but I cancelled that approach because it felt useless. I can redo it if you want me to, it's not hard.